diff --git a/ports/glib/portfile.cmake b/ports/glib/portfile.cmake
index 2b7f9b8b9..4c7d6d406 100644
--- a/ports/glib/portfile.cmake
+++ b/ports/glib/portfile.cmake
@@ -4,7 +4,7 @@ string(REGEX MATCH "^([0-9]*[.][0-9]*)" GLIB_MAJOR_MINOR "${VERSION}")
 vcpkg_download_distfile(GLIB_ARCHIVE
     URLS "https://download.gnome.org/sources/glib/${GLIB_MAJOR_MINOR}/glib-${VERSION}.tar.xz"
     FILENAME "glib-${VERSION}.tar.xz"
-    SHA512 4338bf3e42ccbf3679f60b917194070040ff94ba3643121e8916180d6949f3b8cc308b6ad73912bebcb53a29920954bcfb2216bacca0503473a897f1fd023981
+    SHA512 21176cb95fcab49a781d02789bf21191a96a34a6391f066699b3c20b414b3169c958bd86623deb34ca55912083862885f7a7d12b67cc041467da2ba94d9e83c3
 )
 
 vcpkg_extract_source_archive(SOURCE_PATH
@@ -30,6 +30,10 @@ else()
     list(APPEND OPTIONS -Dlibmount=disabled)
 endif()
 
+if(VCPKG_TARGET_IS_WINDOWS)
+    list(APPEND OPTIONS -Diconv=external)
+endif()
+
 vcpkg_configure_meson(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
@@ -84,14 +88,6 @@ elseif(VCPKG_TARGET_IS_OSX)
 endif()
 vcpkg_copy_tools(TOOL_NAMES ${GLIB_TOOLS} AUTO_CLEAN)
 
-set(pc_replace_intl_path gio glib gmodule-no-export gobject gthread)
-foreach(pc_prefix IN LISTS pc_replace_intl_path)
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/lib/pkgconfig/${pc_prefix}-2.0.pc" "\${prefix}/debug/lib/intl" "-lintl")
-    if(NOT VCPKG_BUILD_TYPE)
-        vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/lib/pkgconfig/${pc_prefix}-2.0.pc" "\${prefix}/lib/intl" "-lintl")
-    endif()
-endforeach()
-
 vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/lib/pkgconfig/gio-2.0.pc" "\${bindir}" "\${prefix}/tools/${PORT}")
 vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/lib/pkgconfig/glib-2.0.pc" "\${bindir}" "\${prefix}/tools/${PORT}")
 if(NOT VCPKG_BUILD_TYPE)
@@ -119,6 +115,4 @@ file(REMOVE_RECURSE
     "${CURRENT_PACKAGES_DIR}/share/gdb"
 )
 
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/lib/gio" "${CURRENT_PACKAGES_DIR}/lib/gio")
-
 file(INSTALL "${SOURCE_PATH}/LICENSES/LGPL-2.1-or-later.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
diff --git a/ports/glib/vcpkg.json b/ports/glib/vcpkg.json
index 121b18a96..b4dccaff8 100644
--- a/ports/glib/vcpkg.json
+++ b/ports/glib/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "glib",
-  "version": "2.75.1",
+  "version": "2.74.1",
   "description": "Portable, general-purpose utility library.",
   "homepage": "https://developer.gnome.org/glib/",
   "license": "LGPL-2.1-or-later",
diff --git a/versions/baseline.json b/versions/baseline.json
index f419895e3..ddb8a0c8f 100644
--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2677,7 +2677,7 @@
       "port-version": 0
     },
     "glib": {
-      "baseline": "2.75.1",
+      "baseline": "2.74.1",
       "port-version": 0
     },
     "glibmm": {
diff --git a/versions/g-/glib.json b/versions/g-/glib.json
index cde8dc073..5b1601f26 100644
--- a/versions/g-/glib.json
+++ b/versions/g-/glib.json
@@ -1,15 +1,5 @@
 {
   "versions": [
-    {
-      "git-tree": "8f7840ed02d20169b1610123b0f35463c81c523d",
-      "version": "2.75.1",
-      "port-version": 0
-    },
-    {
-      "git-tree": "d3b428570b820ec1fbedbb37a651c1e1ddcbb91f",
-      "version": "2.75.0",
-      "port-version": 0
-    },
     {
       "git-tree": "62e6236281de8dd6bd983d4d51721374e5559c7c",
       "version": "2.74.1",
